mockwebserver/src/main/java/okhttp3/mockwebserver/MockWebServer.kt

```diff
@@ -526,6 +526,7 @@ class MockWebServer : ExternalResource(), Closeable {
       }
 
       if (socketPolicy === STALL_SOCKET_AT_START) {
+        dispatchBookkeepingRequest(sequenceNumber, socket)
```

Comment:
How does this relate to the change?

Reply:
One of the new test cases uses this socket policy, and there was a bug where it would never proceed past it because it was peeked only.